examples/src/app/example.mjs

```diff
@@ -218,67 +242,76 @@ class Example extends TypedComponent {
     }
 
     renderPortrait() {
```

Comment:
Best to watch this part with "Hide whitespace" enabled as it's just more indentation for the addition of the `fragment()` and is really only 2 lines of actual change.